common/views/components/PageLayout/PageLayout.js

```diff
@@ -116,32 +116,17 @@ const PageLayout = ({
           {children}
         </div>
         {!hideNewsletterPromo && <NewsletterPromo />}
-        {!hideFooter && (
-          <OpeningTimesContext.Consumer>
-            {openingTimes => (
-              <Footer
-                openingTimes={openingTimes}
-                upcomingExceptionalOpeningPeriods={
-                  openingTimes.upcomingExceptionalOpeningPeriods
-                }
-              />
-            )}
-          </OpeningTimesContext.Consumer>
-        )}
-        {hideFooter && (
-          <noscript>
-            <OpeningTimesContext.Consumer>
-              {openingTimes => (
-                <Footer
-                  openingTimes={openingTimes}
-                  upcomingExceptionalOpeningPeriods={
-                    openingTimes.upcomingExceptionalOpeningPeriods
-                  }
-                />
-              )}
-            </OpeningTimesContext.Consumer>
-          </noscript>
-        )}
+        <OpeningTimesContext.Consumer>
+          {openingTimes => (
+            <Footer
+              hide={hideFooter}
```

Comment:
For better or worse, `Footer` has `extraClasses` – if `hide` just determines whether or not the `is-hidden` class should be added, we could get rid of the prop and conditionally add `is-hidden` in the `extraClasses` instead?

Reply:
Hmm, it has to be conditionally added, but only for the js version, so needs to be done within useEffect

Reply:
ah yeah.